Assets/scss/components/_newfooter.scss

```diff
@@ -0,0 +1,218 @@
+.cmp-new-footer {
+    background-color: $color-grey-silver;
+    padding: 0 toRem(24);
+
+    &__main-container {
+        max-width: 768px;
+        margin: 0 auto;
```

Comment:
@shahidrightpoint Instead of margin: 0 auto; you can use margin-inline:auto; since you don't need spacing from top and bottom.
Cc:- @mathrik  @jvaishnavrp  @msharmaRP